httpclient5/src/main/java/org/apache/hc/client5/http/impl/auth/GGSSchemeBase.java

```diff
@@ -133,9 +131,9 @@ protected GSSManager getManager() {
      * @since 4.4
      */
     protected byte[] generateGSSToken(
-            final byte[] input, final Oid oid, final String serviceName, final String authServer) throws GSSException {
+            final byte[] input, final Oid oid, final String authServer) throws GSSException {
```

Comment:
@careylam Could you please keep the original method signature for the backward compatibility sake?